src/core.jl

```diff
@@ -136,6 +136,21 @@ struct Cross <: Drawable
     range::UnitRange{Int}
 end
 
+
+"""
+    rectangle = Rectangle(p, x, y)
```

Comment:
I prefer to not support `Rectangle(p::Point, x, y)` as the meaning of `x` and `y` can be confusing.

Instead, I prefer the following API:

```julia
Rectangle(from::Point, to::Point)
Rectangle(from::CartesianIndex, to::CaretesianIndex)
```

Reply:
Yes this seems more reasonable. I tried to follow [Matlab's API](https://www.mathworks.com/help/images/ref/images.roi.rectangle.html) but we can do better 😸